scl/azure/azure-monitor.conf

```diff
@@ -49,47 +52,3 @@ block destination _azure_monitor_internal(
     `__VARARGS__`
   );
 };
-
-block destination azure_monitor_builtin(
```

Comment:
Please keep the old destinations for a few more releases because this would be a breaking change otherwise.
We can deprecate the old ones and document only the new one, but let's keep them here for 2 more releases.

Reply:
(I'm sure nobody started using it, so the whole deprecation ceremony is useless, but due to how we version AxoSyslog releases, this is the cleanest and fair way.)